comprobo_final_project/scripts/simulator/robot.py

```diff
@@ -46,10 +55,11 @@ def step(self, step_size):
 
         # Update velocity
         self.pose.velocity.x = twist_r * math.cos(original_velocity_theta +
-                step_size * twist_theta)
+                twist_theta / step_freq)
```

Comment:
i'm not sure if these new mathematical calculations make sense, I'd say double check, but if you're sure then I'm good with them.